code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -192,11 +201,15 @@ Please contact me on #coderbus IRC. ~Carnie x
 
 /mob/living/carbon/human/update_inv_wear_id()
 	remove_overlay(ID_LAYER)
+
 	if(wear_id)
 		if(client && hud_used && hud_used.hud_shown)
 			wear_id.screen_loc = ui_id
 			client.screen += wear_id
 
+		if(dna && istype(dna.species, /datum/species/pony))

```

Comment:
i sure hope not but then again we also said lizards weren't ever going to not be admin only and that genetic catbeast ears wouldn't ever be enabled by default